src/event-handler.js

```diff
@@ -1,94 +1,108 @@
-import Composer from 'telegraf/composer.js';
+import Composer from 'telegraf';
 import { apiService } from './dependencies.js';
 
 export const botUseHandler = (ctx) => {
-	if (ctx.update.message.text === '/create') {
-		ctx.replyWithMarkdown(
-			`Щоб розпочати роботу, пришли мені опис, фото та локацію місця події. Роби все послідовно, тобто спочатку опис, потім фото, а в кінці локація.\n\n*Чекаю від тебе опис місця.*`
-		);
-		return ctx.wizard.selectStep(2);
-	} else {
-		ctx.replyWithMarkdown(
-			`Привіт, мене звати CheClean. Я створений для допомоги нашому місту 😊 \nДля *початку* роботи натисни на команду /create \nДля *допомоги* використай команду /help \nДля *відміни* натисни на команду /cancel`
-		);
-		return ctx.wizard.next();
-	}
+  if (ctx.update.message.text === '/create' && ctx.update.callback_query.data === 'create') {
+    ctx.replyWithMarkdown(
+      `Щоб розпочати роботу, пришли мені опис, фото та локацію місця події. Роби все послідовно, тобто спочатку опис, потім фото, а в кінці локація.\n\n*Чекаю від тебе опис місця.*`,
```

Comment:
Barely same messages here. Please, move this message in it's own variable and replace it

Reply:
Yes, I see a problem. Solved it in the following commits. Thank you.